src/CssVariables/index.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import * as theme from '../theme.js'
+
+/**
+ * Maps over each theme value in a section and converts it to a css custom property (variable)
+ * definition. It uses the export name as a prefix, and suffixes that with the key.
+ */
+const renderThemeSection = ([sectionKey, section]) => {
+    const variables = Object.entries(section)
+
+    return variables.map(([ varKey, varVal ]) => (`--${sectionKey}-${varKey}: ${varVal};`)).join('\n')
+}
+
+/**
+ * Maps over the exported sections of the theme
+ */
+const renderTheme = () => {
+    const sections = Object.entries(theme)
```

Comment:
Can we make this a parameter? Then all the functions, except the component are 100% pure (and the theme could eventually be made a parameter of the `CssVariables` component (if we ever want that)

Reply:
That sounds like a good idea. That way it's just a generic component that injects css variables, and it's not tied too much to our implementation. So people could also use it to inject some extra variables while they're at it for example.